src/main/rx/crsf.c

```diff
@@ -146,13 +145,12 @@ typedef struct crsfPayloadLinkstatistics_s {
     uint8_t Downlink_RSSI;
     uint8_t Downlink_Link_quality;
     uint8_t Downlink_SNR;
+}crsfLinkStatistics_t;
```

Comment:
If I don't say this, someone else will:
`} crsfPayloadLinkStatistics_t;`

Reply:
From this, you could also
`#define CRSF_FRAME_LINK_STATISTICS_PAYLOAD_SIZE (sizeof(crsfLinkStatistics_t))`

Reply:
CRSF_FRAME_LINK_STATISTICS_PAYLOAD_SIZE was already in crsf_protocal.h

Reply:
fixed sp(s)